docs/documentation/upgrading/topics/keycloak/changes-23_0_0.adoc

```diff
@@ -154,3 +154,6 @@ some user profile classes and some validator related classes (but not builtin va
 `keycloak-server-spi` module. However, the packages for java classes remain the same. You might be affected in some corner cases, such as when you
 are overriding the built-in implementation with your own `UserProfileProvider` implementation However, note that `UserProfileProvider` is an unsupported SPI.
 
+= Removal of the `storage` configuration setting
+
+Since this release, it is no longer possible to use `--storage` related CLI options.
```

Comment:
```suggestion
Since this release, it is no longer possible to use `--storage` related CLI options due to the announced discontinuation of the Map Store. 
```

IMHO, it might be good to specify more the reason behind it. I don't think we need to reference a link to the official announcement here as only the keywords might be sufficient. 

@vramik WDYT?

Reply:
Updated accordingly.